app/parse.py

```diff
@@ -16,7 +31,132 @@ class Product:
 
 
 def get_all_products() -> None:
-    pass
+    pages = [
+        {"url": "", "name": "home"},
+        {"url": "computers", "name": "computers"},
+        {"url": "phones", "name": "phones"},
+        {"url": "computers/laptops", "name": "laptops"},
+        {"url": "computers/tablets", "name": "tablets"},
+        {"url": "phones/touch", "name": "touch"}
+    ]
+
+    options = Options()
+    service = Service("path/to/chromedriver")
+    driver = webdriver.Chrome(service=service, options=options)
+
+    for page in pages:
+        product_list = []
+        url = urljoin(HOME_URL, page["url"])
+
+        driver.get(url)
+        time.sleep(2)
+
+        accept_cookies_button = driver.find_elements(
+            By.CSS_SELECTOR, ".acceptCookies"
+        )
+        if accept_cookies_button:
+            accept_cookies_button[0].click()
+            time.sleep(2)
+
+        if page["name"] == "home" or page["name"] == "computers":
+            scrape_page(driver, product_list, page["name"])
+        elif page["name"] == "laptops":
+            laptops_url = urljoin(
+                BASE_URL, "test-sites/e-commerce/more/computers/laptops"
+            )
+            scrape_page(driver, product_list, page["name"], laptops_url)
+        elif page["name"] == "tablets":
+            tablets_url = urljoin(
+                BASE_URL, "test-sites/e-commerce/more/computers/tablets"
```

Comment:
You can add it also to dict as additional_url and than all code will be the same